courses/free-courses-en.md

```diff
@@ -125,6 +125,7 @@
 * [IIT Bombay Foundation of Data Structures (CS213.1x)](https://courses.edx.org/courses/course-v1:IITBombayX+CS213.1x+1T2017/course/)
 * [Intro to Data Structures and Algorithms](https://www.udacity.com/course/data-structures-and-algorithms-in-python--ud513) - Brynn Claypoole, Horatio Thomas (Udacity)
 * [JavaScript Algorithms and Data Structures](https://www.youtube.com/playlist?list=PLC3y8-rFHvwjPxNAKvZpdnsr41E0fCMMP) - Codevolution
+* [Linked Lists Course](https://www.youtube.com/watch?v=Hj_rA0dhr2I) - FreeCodeCamp
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

Reply:
Firstly the link you provided doesn't work, at least for me. Also, I have edited it @eshellman

Reply:
whoops! https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md